common/src/main/java/com/wynntils/models/raid/RaidModel.java

```diff
@@ -221,6 +250,22 @@ public RaidKind getCurrentRaid() {
         return currentRaid;
     }
 
+    public List<String> getRaidBuffMajorIds(String playerName) {
```

Comment:
```suggestion
    public List<String> getMajorIdRaidBuffs(String playerName) {
```

I think this wording makes more sense.

Reply:
This method does return the list of Major Ids though not the list of Raid buffs

Reply:
`getRaidMajorIds`?

Reply:
Resolved